frontend/src/pages/NewRun.tsx

```diff
@@ -309,28 +372,35 @@ class NewRun extends Page<{}, NewRunState> {
     this.setState({ [name]: value, } as any, () => { this._validate(); });
   }
 
-  /* This function is passed as a callback to the PipelineSelector dialog. */
-  protected _pipelineSelectionChanged(selectedId: string): void {
-    this.setState({ unconfirmedDialogPipelineId: selectedId });
+  /* This function is passed as a callback to the experiment selector dialog. */
+  protected _experimentSelectionChanged(selectedExperiment: ApiExperiment): void {
```

Comment:
nit: Can probably inline the setState call in the component prop?

Reply:
Done